server/src/main/java/org/apache/cassandra/sidecar/acl/authorization/BasicPermissions.java

```diff
@@ -70,9 +70,12 @@ public class BasicPermissions
     public static final Permission READ_SCHEMA = new DomainAwarePermission("SCHEMA:READ", CLUSTER_SCOPE);
     public static final Permission READ_SCHEMA_KEYSPACE_SCOPED = new DomainAwarePermission("SCHEMA:READ", KEYSPACE_SCOPE);
     public static final Permission READ_GOSSIP = new DomainAwarePermission("GOSSIP:READ", CLUSTER_SCOPE);
+    public static final Permission WRITE_GOSSIP = new DomainAwarePermission("GOSSIP:WRITE", CLUSTER_SCOPE);
```

Comment:
naming is hard. I do not like `write` as the name, since the permission is to permit whether gossip can start/stop by the client. On the other hand, I do not have suggestions. So feel free to ignore.

Reply:
How about `GOSSIP:CONTROL` or `GOSSIP:MODIFY`. Similarly we can use for Native permission too `NATIVE:CONTROL` or `NATIVE:MODIFY` ?